api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.7.0"
+   "version": "v1.6.0"
```

Comment:
@caesarxuchao Thanks a lot for the tip :) !! So no with pre-commit hook it forces me to make the openapi changes before I make any commit on generated code changes. But now I am actually going to try without hook, let's see how that goes :). Agree may not be a good style, but honestly with hooks it's taking so much time specially when we have multiple commits in a single PR.